### ISSUE TITLE
Naming convention sniffs over-zealous ?

### ISSUE THREAD

**Issue by jrfnl** [WordPress-Coding-Standards/WordPress-Coding-Standards#502]
The naming convention sniffs introduced in #500 seem to me to be a bit over-zealous.

The handbook doesn't mention anything about private properties needing to start with an underscore.

And to be quite fair, IMHO, that shouldn't become part of the standard either as that is an outdated PHP4 practice from when you couldn't indicate the visibility of a property.

Ref: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions

Oh and there is a message error on line 113:
```php
$error = 'Variable "%s" is not in valid camel caps format';
```

should probably read:
```php
$error = 'Variable "%s" is not in valid snake_case format';
```

**Comment by JDGrimes** [comment]
I agree, I don't think that needs to be part of the standard. We could remove that part of the sniff, or we could just disable those particular error messages by default (allowing those that want to use them to re-enable them).

@westonruter, what are your thoughts?

**Comment by westonruter** [comment]
Disabling by default sounds good to me. But since it's all copied from the
Squiz standard removing them isn't going to hurt my feelings either :-)

**Comment by JDGrimes** [comment]
Maybe we should just remove it, if for no other reason that that it would be added maintenance for something that we don't really use. So @westonruter, if you follow that rule and want to use this sniff and maintain it going forward, I'm fine with just disabling that part by default. Otherwise I'd prefer we remove it. Make sense?

**Comment by jrfnl** [comment]
:+1:

**Comment by westonruter** [comment]
See https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/506
